shell/app-shell/elements/shell-stores.js

```diff
@@ -9,17 +9,17 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 */
 
 import Xen from '../../components/xen/xen.js';
-import ArcsUtils from '../lib/arcs-utils.js';
+import ArcsUtils from '../../lib/arcs-utils.js';
 import './arc-store.js';
 import {FbStore} from './fb-data/FbStore.js';
 
 // templates
 const template = Xen.html`
-  <arc-store arc="{{arc}}" data="{{themeData}}" options="{{themeStoreOptions}}" on-change="_onShellThemeChange"></arc-store>
+  <!-- <arc-store arc="{{arc}}" data="{{themeData}}" options="{{themeStoreOptions}}" on-change="_onShellThemeChange"></arc-store>
```

Comment:
Here and below in this file, are these commented out intentionally?

Reply:
shell-stores itself is deprecated. These comments are vestiges of me deprecating it piece-meal until I was finally able to detach the whole thing. It does look sloppy. I guess I don't like deleting things until after some grace-period; maybe this is a bad habit.